### ISSUE TITLE
setup-build-env adds duplicate source on google-cloud-sdk.list

### ISSUE THREAD

**Issue by dashwood8691** [f5devcentral/f5-bigip-image-generator#13]
**Describe the bug**
Runing setup-build-env multiple times on the same box results in duplicate source definition in google-cloud-sdk.list as follows. Running `apt-get update` results in multiple warnings indicating the duplicates.

```bash
# cat /etc/apt/sources.list.d/google-cloud-sdk.list
deb http://packages.cloud.google.com/apt cloud-sdk-bionic main
deb http://packages.cloud.google.com/apt cloud-sdk-bionic main
```

```bash
# apt-get update
Hit:1 http://ftp.riken.go.jp/Linux/ubuntu bionic InRelease
Hit:2 http://ftp.riken.go.jp/Linux/ubuntu bionic-updates InRelease
Hit:3 http://ftp.riken.go.jp/Linux/ubuntu bionic-backports InRelease
Hit:4 http://ftp.riken.go.jp/Linux/ubuntu bionic-security InRelease
Hit:5 http://packages.cloud.google.com/apt cloud-sdk-bionic InRelease
Reading package lists... Done
W: Target Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Translations (main/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Translations (main/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target CNF (main/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target CNF (main/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Translations (main/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target Translations (main/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target CNF (main/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
W: Target CNF (main/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/google-cloud-sdk.list:1 and /etc/apt/sources.list.d/google-cloud-sdk.list:2
```

**To Reproduce**
Simply run the script `setup-build-env` multiple times.

**Expected behavior**
It's very possible for users to run the script multiple times, so it should check if the line already exists and avoid adding it when it does. A conditional like below will help users to avoid the issue.

```
     # Add the Cloud SDK distribution URI as a package source:
-    echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" \
-            | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+    if ! grep -Fxq "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" \
+        /etc/apt/sources.list.d/google-cloud-sdk.list
+    then
+        echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" \
+              | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+    fi
```

**Additional context**
I've been using the tool since v1.3, and updating to a newer version requires running `setup-build-env` to set up environment for the version. Following the required procedure results in hitting the issue.

I'm on v1.5 currently.

```bash
# ./build-image -v
Initializing global variable definitions
version=1.5
Log file has not been set. Cannot collect snapshots.
```

**Comment by kmaloney** [comment]
I was able to reproduce this issue.  This issue is being tracked internally with Jira bug VECLOUD-963.

**Comment by kmaloney** [comment]
Resolved in v1.6.
